Requests/Builders/CreateMemberRequestBuilder.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pyrus.ApiClient.Requests.Builders
+{
+   public  class CreateMemberRequestBuilder
+    {
+        private readonly CreateMemberRequest _request;
+
+        public CreateMemberRequestBuilder(string email)
+        {
+            _request = new CreateMemberRequest
+            {
+                Email = email
+            };
+        }
+
+		public CreateMemberRequestBuilder WithFirstName(string name)
```

Comment:
The idea of Builder pattern is to provide humand readable method call chain to construct a request.
In this case all method have the same signature witch does not have a lot of value.

Reply:
```
RequestBuilder
    .CreateAccount(login: "someone@domain.com")
        .For("John", "Doe")
        .WorkingAt(itDepartmentId)
        .As("CTO")
        .WorkPhone("+1(800) 123 4567")
        .Skype("john.doe")
        .Process(apiClient)
```